roxie/ccd/ccdmain.cpp

```diff
@@ -464,6 +464,16 @@ static std::vector<std::pair<unsigned, unsigned>> agentChannels;
 
 void *leakChecker = nullptr;   // Used to deliberately leak an allocation to ensure leak checking is working
 
+unsigned __int64 currentTopologyHash = 0;
+unsigned __int64 originalTopologyHash = 0;
+
+hash64_t getTopologyHash()
+{
+    StringBuffer xml;
+    toXML(topology, xml, 0, XML_SortTags);
```

Comment:
Should this somehow exclude lockDali from the hash?  Will that cause any problems?

Reply:
The intention was for this one to include lockDali, and the other one to exclude it, so we can see both